src/server/server.js

```diff
@@ -130,6 +131,16 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function stop() {
+    simSocket.closeConnections();
+
+    config.newInstance();
+    plugins.clear();
+    simFiles.clear();
+    prepare.clear();
```

Comment:
Maybe `reset` is a more indicative name for these functions (as opposed to `clear`).